class.jetpack.php

```diff
@@ -5363,8 +5363,20 @@ function allow_wpcom_public_api_domain( $domains ) {
 		return $domains;
 	}
 
+	static function is_redirect_encoded( $redirect_url ) {
+		return wp_startswith( $redirect_url, 'https%3A%2F%2F' );
```

Comment:
The `redirect_url` is (as far as I know) public-api.wordpress.com and so should be set by us and never http, but added the extra checks just in case!